packages/rocketchat-lib/server/models/Rooms.js

```diff
@@ -127,8 +127,8 @@ class ModelRooms extends RocketChat.models._Base {
 					return item._room;
 				}
 				console.log('Empty Room for Subscription', item);
-				return {};
 			});
+			data = data.filter(item => item);
```

Comment:
this is weird to see, but it works :joy:

Reply:
I suppose returning null evaluates to false during the filter resulting in null being excluded? 

Very odd looking indeed 😁